source/auth/auth.rst

```diff
@@ -1125,6 +1125,9 @@ Q: Why does SCRAM sometimes SASLprep and sometimes not?
 Version History
 ===============
 
+Version 1.8.2 Changes
```

Comment:
So we should also bump the spec version on line 8 to 1.8.2, right?

Reply:
Ah, yes. Done.